### PR TITLE
docs: Add example usage of reconciliationPaused feature

### PR DESCRIPTION
**Pull request by sbernauer**

### CODE REVIEW

File: modules/concepts/pages/operations/cluster_operations.adoc [hunk 1]
```diff
@@ -9,6 +9,13 @@ This is useful when updating operators, debugging or testing of new settings:
 
 If not specified, `clusterOperation.reconciliationPaused` and `clusterOperation.stopped` default to `false`.
 
+IMPORTANT: In case you create a stacklet and set `reconciliationPaused` to `true` during the creation, no resources will be created.
```
Comment (adwk67) [inline modules/concepts/pages/operations/cluster_operations.adoc:12]:
```suggestion
IMPORTANT: If you create a stacklet and set `reconciliationPaused` to `true` during the creation, no resources will be created.
```

File: modules/concepts/pages/operations/cluster_operations.adoc [hunk 2]
```diff
@@ -9,6 +9,13 @@ This is useful when updating operators, debugging or testing of new settings:
...
+This is due to the fact that you basically state "Hey my-operator, here is an MyCluster, but please ignore it".
```
Comment (adwk67) [inline modules/concepts/pages/operations/cluster_operations.adoc:13]:
```suggestion
This is due to the fact that you basically state "Hey, my-operator, here is a MyCluster, but please ignore it".
```

**Review by adwk67 — CHANGES_REQUESTED** [review]
Just very minor things.

**Review by razvan — DISMISSED** [review]

**Review by adwk67 — DISMISSED** [review]

File: modules/concepts/pages/operations/cluster_operations.adoc [hunk 3]
```diff
@@ -9,6 +9,13 @@ This is useful when updating operators, debugging or testing of new settings:
 
 If not specified, `clusterOperation.reconciliationPaused` and `clusterOperation.stopped` default to `false`.
 
+IMPORTANT: If you create a stacklet and set `reconciliationPaused` to `true` during the creation, no resources will be created.
+This is due to the fact that you basically state "Hey, my-operator, here is a MyCluster, but please ignore it".
```
Comment (NickLarsenNZ) [inline modules/concepts/pages/operations/cluster_operations.adoc:13]: I would reword it less casually. Will come back with a suggestion soon.

**Review by NickLarsenNZ — CHANGES_REQUESTED** [review]
Will complete the review soon, and also provide a suggestion

File: modules/concepts/pages/operations/cluster_operations.adoc (hunk 3 above)
Comment (NickLarsenNZ) [inline modules/concepts/pages/operations/cluster_operations.adoc:13]:
```suggestion
[IMPORTANT]
===========
When `reconciliationPaused` is set to `true`, operators will ignore reconciliation events (creations, updated, deletions).

Furthermore, if you create a stacklet where `reconciliationPaused` is set to `true`, no resources will be created.
===========
```

**Review by NickLarsenNZ — APPROVED** [review]
Added my suggestion

File: modules/concepts/pages/operations/cluster_operations.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbernauer) [reply modules/concepts/pages/operations/cluster_operations.adoc]: I did go with @NickLarsenNZ suggestion, hope that works for you

File: modules/concepts/pages/operations/cluster_operations.adoc [hunk 4]
```diff
@@ -9,6 +9,17 @@ This is useful when updating operators, debugging or testing of new settings:
...
+[IMPORTANT]
+===========
+When `reconciliationPaused` is set to `true`, operators will ignore reconciliation events (creations, updates, deletions).
+
+Furthermore, if you create a stacklet where `reconciliationPaused` is set to `true`, no resources will be created.
+===========
+
+IMPORTANT: When setting `clusterOperation.reconciliationPaused` and `clusterOperation.stopped` to true in the same step, `clusterOperation.reconciliationPaused` will take precedence.
+This means the cluster will stop reconciling immediately and the `stopped` field is ignored.
+To avoid this, the cluster should first be stopped and then paused.
```
Comment (NickLarsenNZ) [inline modules/concepts/pages/operations/cluster_operations.adoc:21]: My apologies, [blocks need (?) to use _four_ `=`](https://docs.antora.org/antora/latest/asciidoc/examples/#delimited). I thought it just had to match :shrug: 

Also, I forgot to delete the previous text in the suggestion.

```suggestion
====
When `reconciliationPaused` is set to `true`, operators will ignore reconciliation events (creations, updates, deletions).

Furthermore, if you create a stacklet where `reconciliationPaused` is set to `true`, no resources will be created.
====
```

**Review by NickLarsenNZ — CHANGES_REQUESTED** [review]
Sorry, my suggestion had errors

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

File: modules/concepts/pages/operations/cluster_operations.adoc (hunk 4 above)
Comment (sbernauer) [reply modules/concepts/pages/operations/cluster_operations.adoc]: Applied in 688a3e5c

**Review by NickLarsenNZ — APPROVED** [review]
LGTM
